src/ParseServer.js

```diff
@@ -403,6 +420,23 @@ class ParseServer {
     config: LiveQueryServerOptions,
     options: ParseServerOptions
   ) {
+    if (config) {
+      if (Object.prototype.toString.call(config) === '[object Object]') {
+        Config.validateConfigKeyNames(
+          Object.keys(LiveQueryServerOptionsDef),
+          Object.keys(config),
+          'LiveQueryServerOptions'
+        );
+
+        if (Config.failedConfigKeyVerification) {
+          delete Config.failedConfigKeyVerification;
```

Comment:
I just noticed; is the verification result set on the config object itself? We should not do that. I have seen that this is done in other cases as you pointed out with the "internal" keys, but that can be problematic for various reasons. Why don't you just throw inside the `Config.validateConfigKeyNames` method? This way you could also avoid the code duplication, as we see the error message is the same 2 times.

Reply:
That was just so that all the key names are checked before an error is thrown otherwise it will stop the execution after the first wrong key is detected, and in one of your previous comments I remember you mentioned that 2 warnings should be logged if 2 keys are wrong.

Reply:
Yes, seeing all wrong keys at once is definitely better. But we also don't want code duplication or modifying the config object.

Reply:
You are right we don't need to throw error message from 2 different places. And for config variable this only gets set if there is an error in one of the keys, otherwise everything will be as it was.

Reply:
Any updates on this PR? Are you still considering this?

Reply:
Sure, I was just waiting for you to make the changes you suggested

Reply:
Is this ready for review?

Reply:
Yes. Sorry was travelling so couldn't respond.